src/container/tests/containerSpec.js

```diff
@@ -536,13 +536,12 @@ describe('Container', () => {
                 }
             );
 
-            container.get('exampleService');
-
-            extraHandler.onGetComplete.should.have.been.calledWith(
-                'extra',
-                sinon.match.instanceOf(ExtensionApi)
-            );
-
+            return container.get('exampleService').then(() => {
```

Comment:
This is now misleading. `onGetComplete` should trigger immediately, independent of promise resolution.